chromium_src/chrome/browser/resources/side_panel/customize_chrome/app.html.ts.lit_mangler.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { mangle } from 'lit_mangler'
+
+// Update "Customize toolbar" button's icon
+mangle(
+  (element) => {
+    const el = element.querySelector('#toolbarButton')
+    if (!el) {
+      throw new Error('[Customize Chrome] #toolbarButton is gone.')
+    }
+
+    // Remove existing chevron icon
+    el.querySelector('cr-icon[icon="cr:chevron-right"]')?.remove()
+
+    // Insert new icon at the start of the button
+    el.insertAdjacentHTML(
+      'afterbegin',
+      `<leo-icon name="window-edit" slot="prefix-icon"></leo-icon>`,
+    )
```

Comment:
As `cr-icon` is wrapping a `leo-icon` I think you might be able to just pass the leo icon name straight through.
```suggestion
    el.querySelector('cr-icon[icon="cr:chevron-right"]')?.setAttribute("icon", "window-edit")
```

Reply:
Probably worth checking the icon exists, in case they change it in the future :smile:

Reply:
Thanks @fallaciousreasoning ! That's more declartive. Done